src/ui.form-checkbox/index.vue

```diff
@@ -223,6 +230,11 @@ onMounted(() => {
   checkboxName.value = isCheckboxInGroup.value ? checkboxGroupName : props.name;
 });
 
+watchEffect(
+  () => (checkboxColor.value = getCheckboxGroupColor ? getCheckboxGroupColor() : props.color),
```

Comment:
and you can use `toValue(checkboxGroupColor)` and same in other places, please follow the same way of working with provide / inject in all library.